HomeScreenLib/src/main/java/com/example/homescreenlib/ui/HomeActivity.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.homescreenlib.ui
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.databinding.DataBindingUtil
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProvider
+import com.example.homescreenlib.R
+import com.example.homescreenlib.data.DashboardItems
+import com.example.homescreenlib.databinding.ActivityHomeBinding
+import com.example.homescreenlib.viewmodel.DashboardViewModel
+import com.example.homescreenlib.viewmodel.ViewModelFactory
+
+class HomeActivity : AppCompatActivity() {
```

Comment:
make HomeActivity as internal

Reply:
Getting Error Cannot access 'HomeActivity': it is internal in 'com.example.homescreenlib.ui'
I created an Intent in App module to redirect to HomescreenLib.